docs/release_notes/0.45.0.md

```diff
@@ -0,0 +1,20 @@
+# Release 0.45.0
+
+## Features
+
+- Get nodegroup desired capacity from autoscaling instead of cloudformation (#3515)
```

Comment:
this gets modified again in https://github.com/weaveworks/eksctl/pull/3543 to include min & max capacity

Reply:
lol i thought 3543 looked familiar but didn't know if i was imagining it 😂 

okay when the other one is merged i will swap this line